src/compounds/ActionsGroup/actions/SendMessage.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useRef } from 'react'
+import {
+  Button,
+  FloatingLabel,
+  Form,
+  Modal,
+} from 'react-bootstrap'
+import './styles.scss'
+
+const SendMessage = ({ onSubmit, handleClose }) => {
+  const inputRef = useRef(null)
+
+  const handleSubmit = (event) => {
+    event.preventDefault()
+    onSubmit({ value: inputRef.current.value })
+    handleClose()
+  }
+
+  return (
+    <Modal show={true} onHide={handleClose}>
+      <Modal.Header closeButton>
+        <Modal.Title>Send a Message</Modal.Title>
+      </Modal.Header>
+      <Form>
+        <Modal.Body>
+          <Form.Group>
+            <FloatingLabel controlId='send-message' className='mb-3' label='Message'>
+              <Form.Control as='textarea' placeholder='Message' ref={inputRef} />
```

Comment:
this placeholder doesn't actually show in the text area. what you see is the control name. the [docs](https://react-bootstrap.netlify.app/forms/floating-labels/#example) say it's required though.